src/core/ColorPicker/ColorPicker.tsx

```diff
@@ -0,0 +1,62 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { useTheme } from '../utils/hooks/useTheme';
+import '@itwin/itwinui-css/css/color-picker.css';
+import { CommonProps } from '../utils/props';
+
+export const DefaultColors = [
+  { color: '#FFFFFF', name: 'WHITE' },
+  { color: '#5A6973', name: 'GREY' },
+  { color: '#00121D', name: 'KURETAKE BLACK MANGA' },
+  { color: '#002A44', name: 'RHAPSODY IN BLUE' },
+  { color: '#00426B', name: 'DARK IMPERIAL BLUE' },
+  { color: '#005A92', name: 'JETSKI RACE' },
+  { color: '#0073BA', name: 'FRENCH BLUE' },
+  { color: '#008BE1', name: 'BLUE COLA' },
+  { color: '#30B0FF', name: 'FANTASY CONSOLE SKY' },
+  { color: '#58BFFF', name: 'HELLO SUMMER' },
+  { color: '#7FCEFF', name: 'CHROMIS DAMSEL BLUE' },
+  { color: '#A6DDFF', name: 'DROPLET' },
+  { color: '#CDECFF', name: 'LUCID DREAMS' },
+  { color: '#E5F5FD', name: 'KODAMA WHITE' },
+  { color: '#010200', name: 'REGISTRATION BLACK' },
+  { color: '#122306', name: 'YUZU SOY' },
+  { color: '#23450b', name: 'FOREST GREEN' },
+  { color: '#346711', name: 'TATZELWURM GREEN' },
+  { color: '#458816', name: 'CHLOROPHYLL' },
+  { color: '#56AA1C', name: 'PLASTIC PINES' },
+  { color: '#5FBB1F', name: 'FIELD GREEN' },
+  { color: '#67CC22', name: 'GREEN HIGH' },
+  { color: '#91E458', name: 'LILLIPUTIAN LIME' },
+  { color: '#B2EC8B', name: 'GREEN DAY' },
+  { color: '#D4F4BD', name: 'TEA GREEN' },
+  { color: '#EEF6E8', name: 'VERDE PASTEL' },
+  { color: '#9BA5AF', name: 'SERYI GREY' },
+  { color: '#CF0000', name: 'RED EPIPHYLLUM' },
+  { color: '#FF6300', name: 'SAFETY ORANGE' },
+  { color: '#FFC335', name: 'RISE-N-SHINE' },
+];
+
+export type ColorPickerProps = {
+  /**
+   * Any custom nodes that will be added within the color palette.
```

Comment:
Here is not any custom nodes.. Maybe mention, we expect `ColorSwatch` components in here.

Reply:
updated the comment